helm/polaris/templates/_helpers.tpl

```diff
@@ -150,18 +150,18 @@ line breaks, they will be escaped and a multi-line option will be printed.
 {{- $valAsString = $valAsString | nindent 4 | replace "\n" "\\\n" -}}
 {{- end -}}
 {{- end -}}
-{{ print $key "=" $valAsString }}
+{{ print (include "polaris.escapeConfigOptionKey" $key) "=" $valAsString }}
 {{- end -}}
 
 {{/*
-Convert a dict into a string formed by a comma-separated list of key-value pairs: key1=value1,key2=value2, ...
+Escapes a property key to be used in a configmap, conforming with the Java parsisng rules for
```

Comment:
We were missing until now a proper escape mechanism for config option keys. Before this change, any realm containing a key termination character such as a space or `=`, would produce invalid configuration.